sdk/core/Azure.Core.Experimental/src/JsonData.cs

```diff
@@ -65,7 +76,7 @@ public JsonData(object? value) : this(value, DefaultJsonSerializerOptions)
         /// <param name="value">The value to convert.</param>
         /// <param name="options">Options to control the conversion behavior.</param>
         /// <param name="type">The type of the value to convert. </param>
-        public JsonData(object? value, JsonSerializerOptions options, Type? type = null)
+        internal JsonData(object? value, JsonSerializerOptions options, Type? type = null)
```

Comment:
Would we want to also expose a way to use this with Azure.Core's ObjectSerializer?